app-containers/containers-common/containers-common-0.57.3.ebuild

```diff
@@ -0,0 +1,59 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="Common config files and docs for Containers stack"
+HOMEPAGE="https://github.com/containers/common"
+
+if [[ ${PV} == 9999* ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/containers/common.git"
+else
+	SRC_URI="https://github.com/containers/common/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+	S="${WORKDIR}/${P#containers-}"
+	KEYWORDS="~amd64 ~arm64 ~riscv"
+fi
+
+LICENSE="Apache-2.0"
+SLOT="0"
+RESTRICT="test"
+RDEPEND="
+	app-containers/containers-image
+	app-containers/containers-storage
+	app-containers/containers-shortnames
+	net-firewall/nftables
```

Comment:
Maybe it should still block `!<app-containers/podman-4.5.0-r1` which installed seccomp.json prior to 275aeff423d63b6cdde5b508f59d5af5248f16c0.

Reply:
but then pkgcheck warns 
`NonexistentBlocker: version 5.29.2: nonexistent blocker RDEPEND="!<app-containers/podman-4.5.0-r1": no matches in repo history`

which makes sense to me. Will there ever be case where someone has (very old) podman-4.5.0-r1 and trying to install (latest) containers-common-0.57.3 ?!

Reply:
Since podman-4.5.0 was released Apr 14 2023, if someone tries to update a system that hasn't been updated since them, then containers-common-0.57.3 will get pulled in as a dependency of the new podman. Since containers-common-0.57.3 needs to install before the new version of podman, it will trigger a file collision.

Seems like something is wrong with pkgcheck since I see a cutoff of 2 years here:
https://github.com/pkgcore/pkgcheck/blob/master/src/pkgcheck/checks/metadata.py#L1059

Reply:
> Since podman-4.5.0 was released Apr 14 2023, if someone tries to update a system that hasn't been updated since them, then containers-common-0.57.3 will get pulled in as a dependency of the new podman. Since containers-common-0.57.3 needs to install before the new version of podman, it will trigger a file collision.

Wow beautifully explained. Now I got it. But for how long we should take care of such old users? 2years?

Reply:
@zmedico  okay. After looking at https://github.com/gentoo/gentoo/commit/fc2204a2325f3feaff80d6195c307135de462811 I think collision won't occur because podman-4.5.0 installed policy.json at _/etc/containers/policy.json.example_ and default.yaml wasn't even installed. New containers-common/image installs it at _/etc/containers/policy.json_ so problem solved? No further action needed?

Reply:
Ah, ok, sure that works. Thanks for checking!

Reply:
time to merge! and could you also take care of https://bugs.gentoo.org/921771 then https://github.com/gentoo/gentoo/pull/34744 ?